platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/BasicConfig.java

```diff
@@ -72,21 +68,19 @@
  */
 @ConfigData
 public record BasicConfig(
-        @ConfigProperty(defaultValue = "true") boolean showInternalStats,
-        @ConfigProperty(defaultValue = "false") boolean verboseStatistics,
-        @ConfigProperty(defaultValue = "1000") int numConnections,
-        @ConfigProperty(defaultValue = "true") boolean logStack,
-        @ConfigProperty(defaultValue = "500") int sleepHeartbeat,
-        @ConfigProperty(defaultValue = "60") double statsSkipSeconds,
-        @ConfigProperty(defaultValue = "10") int freezeSecondsAfterStartup,
-        @ConfigProperty(defaultValue = "true") boolean loadKeysFromPfxFiles,
-        @ConfigProperty(defaultValue = "1000") int jvmPauseDetectorSleepMs,
-        @ConfigProperty(defaultValue = "1000") int jvmPauseReportMs,
-        @ConfigProperty(defaultValue = "true") boolean enablePingTrans,
-        @ConfigProperty(defaultValue = "1") long pingTransFreq,
-        @ConfigProperty(defaultValue = "60s") Duration hangingThreadDuration,
-        @ConfigProperty(defaultValue = "data/saved") String emergencyRecoveryFileLoadDir,
-        @ConfigProperty(defaultValue = "0") long genesisFreezeTime) {
+        @ConfigProperty(defaultValue = "1000") int numConnections, // core
```

Comment:
Did you mean to remove these trailing comments? Ones that are not referenced should be removed

Reply:
Yes, I did; I removed the comments from the other records and missed this one. Thanks for spotting.
Cool, I'll also remove the unreferenced properties 🧹

Reply:
Done @poulok  
@nathanklick. Please check, given that this introduced a change in settings.txt